src/main/scala/com/codacy/client/bitbucket/client/BitbucketClient.scala

```diff
@@ -1,12 +1,14 @@
 package com.codacy.client.bitbucket.client
 
+import java.util.concurrent.{SynchronousQueue, ThreadPoolExecutor, TimeUnit}
+
 import com.codacy.client.bitbucket.util.HTTPStatusCodes
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/Codacy/bitbucket-scala-client/file/1807008534/issues/source?bid=2474672&fileBranchId=3073963#l5)